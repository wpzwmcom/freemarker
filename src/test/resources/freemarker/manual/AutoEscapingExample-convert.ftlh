<#ftl output_format="HTML"><#-- Needed for the Eclipse plugin -->

<#assign mo1 = "Foo's bar {}"?esc>
HTLM: ${mo1}
XML:  <#outputformat 'XML'>${mo1}</#outputformat>
RTF:  <#outputformat 'RTF'>${mo1}</#outputformat>

<#assign mo2><p>Test</#assign>
HTML: ${mo2}
XML:  <#attempt><#outputformat 'XML'>${mo2}</#outputformat><#recover>Failed</#attempt>
RTF:  <#attempt><#outputformat 'RTF'>${mo2}</#outputformat><#recover>Failed</#attempt>
