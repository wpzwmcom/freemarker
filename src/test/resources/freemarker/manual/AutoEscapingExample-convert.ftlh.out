HTLM: Foo&#39;s bar {}
XML:  Foo&apos;s bar {}
RTF:  Foo's bar \{\}

HTML: <p>Test
XML:  Failed
RTF:  Failed
